app/code/Magento/CatalogGraphQl/etc/schema.graphqls

```diff
@@ -10,6 +10,10 @@ type Query {
         sort: ProductAttributeSortInput @doc(description: "Specifies which attributes to sort on, and whether to return the results in ascending or descending order.")
     ): Products
     @resolver(class: "Magento\\CatalogGraphQl\\Model\\Resolver\\Products") @doc(description: "The products query searches for products that match the criteria specified in the search and filter attributes.") @cache(cacheIdentity: "Magento\\CatalogGraphQl\\Model\\Resolver\\Product\\Identity")
+    product (
```

Comment:
I'm not sure that `Query.product` really makes it clear now that it returns _multiple_ products.

How about something like `Query.productsByID`?